EventsExpress/appsettings.json

```diff
@@ -15,7 +15,8 @@
   },
   "Logging": {
     "LogLevel": {
-      "Default": "Warning"
+      "Default": "Warning",
+      "Microsoft.EntityFrameworkCore": "Debug"
```

Comment:
Move this setting to apsettings.development.json please.

Reply:
Done